src/tests/unittests/algorithms/test_random_telegraph_signal.py

```diff
@@ -76,3 +78,10 @@ def test_create_integer_histogram(self):
 
         with self.assertRaises(Exception):
             _ = _create_integer_histogram([])
+
+    def test_two_level_threshold(data):
+
+        data = generate_RTS_signal(100000, std_gaussian_noise=0.24, rate_up=10e3, rate_down=20e3)
+        result = two_level_threshold(data)
+        plot_two_level_threshold(result, fig=1)
```

Comment:
Because the figure is printed this one should be added to test_random_telegraph_signal.py in subdirectory integration.